catavatar/lang/C/messages.po

```diff
@@ -17,26 +17,34 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: catavatar.php:57
+#: catavatar.php:60
 msgid "Use Cat as Avatar"
 msgstr ""
 
-#: catavatar.php:58
+#: catavatar.php:61
 msgid "More Random Cat!"
 msgstr ""
 
-#: catavatar.php:59
+#: catavatar.php:62
 msgid "Reset to email Cat"
 msgstr ""
 
-#: catavatar.php:61
+#: catavatar.php:64
 msgid "Cat Avatar Settings"
 msgstr ""
 
-#: catavatar.php:103 catavatar.php:112 catavatar.php:121
+#: catavatar.php:100
+msgid "The cat hadn't found itself."
+msgstr ""
+
+#: catavatar.php:109
 msgid "There was an error, the cat ran away."
```

Comment:
Wie viele Katzen werden Selbstfindungsprobleme haben? I like the humor with these cats.